gitlab/v4/objects/projects.py

```diff
@@ -579,6 +579,29 @@ def mirror_pull(self, **kwargs: Any) -> None:
         path = f"/projects/{self.encoded_id}/mirror/pull"
         self.manager.gitlab.http_post(path, **kwargs)
 
+    @cli.register_custom_action("Project")
+    @exc.on_http_error(exc.GitlabCreateError)
```

Comment:
```suggestion
    @exc.on_http_error(exc.GitlabGetError)
```

Reply:
Oh yeah. The dreaded copy-paste error ☹️

Good catch!